src/workflows-app/SubmissionDetails.test.js

```diff
@@ -89,29 +101,29 @@ describe('Submission Details page', () => {
     const table = await screen.findByRole('table');
 
     // Assert
-    expect(table).toHaveAttribute('aria-colcount', '4');
+    expect(table).toHaveAttribute('aria-colcount', '5');
     expect(table).toHaveAttribute('aria-rowcount', '3');
 
     const rows = within(table).getAllByRole('row');
     expect(rows.length).toBe(3);
 
     const headers = within(rows[0]).getAllByRole('columnheader');
-    expect(headers.length).toBe(4);
+    expect(headers.length).toBe(5);
```

Comment:
Not necessary to change (I realize this is matching the pattern of existing code), but FYI, there's a [`toHaveLength`](https://jestjs.io/docs/expect#tohavelengthnumber) matcher.

```suggestion
    expect(headers).toHaveLength(5);
```